index.ts

```diff
@@ -10,12 +10,13 @@ import websocket = require("websocket");
 import bodyParser = require("body-parser");
 import dotenv = require("dotenv");
 import session = require("express-session");
+import {Express} from "express";
 
 dotenv.config();
 
 
-var app = express();
-var server = new http.Server(app);
+var app: Express = express();
```

Comment:
With proper types for the `express` module the type of `app` can be easily inferred here by typescript and is also for the reader, so it could be omitted here.